test/user.js

```diff
@@ -18,6 +18,28 @@ const api = require('./lib/api.js');
 const prompt = require('../lib/prompt.js');
 const user = require('../lib/user.js');
 const util = require('../lib/util.js');
+const constants = require('./../lib/constants');
+
+const invalidEmail = 'invalid@example.com';
```

Comment:
The 'invalidEmail' is used throughout the tests and its value never changes. Could you be more specific as to why you think it should be moved and attached to this?